integration/mediation-tests/tests-platform/tests-rabbitmq/src/test/java/org/wso2/carbon/esb/rabbitmq/utils/RabbitMQTestUtils.java

```diff
@@ -140,4 +143,26 @@ public static void waitForMssagesToGetPublished() throws InterruptedException {
         Thread.sleep(TIME_FOR_MESSAGE_PUBLISH);
     }
 
+    public static void stopRabbitMq() throws IOException, InterruptedException {
+        executeDockerCommand("docker stop " + RABBITMQ_CONTAINER);
+    }
+
+    public static void startRabbitMq() throws IOException, InterruptedException {
+        executeDockerCommand("docker start " + RABBITMQ_CONTAINER);
+    }
+
+    private static void executeDockerCommand(String command) throws IOException, InterruptedException {
+        log.info("Executing docker command: " + command);
+        Process process = Runtime.getRuntime().exec(command);
+        BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
+
+        String line;
+        while ((line = reader.readLine()) != null) {
+            log.info(line);
+        }
+
+        int exitCode = process.waitFor();
+        log.info("Command execution exited with error code : " + exitCode);
```

Comment:
is this error code or just code

Reply:
It logs the exit code, not essentially an error code. Will fix in the next PR.